screens/welcome.js

```diff
@@ -1,92 +1,120 @@
-import React, { Fragment, Component } from 'react';
+import React, { Fragment, Component } from "react";
 import {
   SafeAreaView,
   StyleSheet,
   ScrollView,
   View,
   Text,
   StatusBar,
-  Button
-} from 'react-native';
+  Button,
+  AsyncStorage
+} from "react-native";
 
-export default class Welcome extends React.Component{
+export default class Welcome extends React.Component {
+  state = {
+    onboarded: false
+  };
+
+  componentDidMount() {
+    this.isUserOnboarded();
+  }
+  clearAsyncStorage = async () => {
+    AsyncStorage.clear();
+  };
+
+  isUserOnboarded = async () => {
+    const name = await AsyncStorage.getItem("Nickname");
+    if (name != null) {
+      //nickname found
+      this.setState({ onboarded: true });
+    }
+  };
 
   render() {
     const { navigation } = this.props;
     console.disableYellowBox = true;
+    const onboarded = this.state.onboarded;
+    let button;
+    if (!onboarded) {
+      button = (
+        <Button
+          icon={{
+            name: "arrow-forward",
+            size: 15,
+            color: "white"
+          }}
+          title="Get started"
+          onPress={() => navigation.navigate("Intro")}
+        />
+      );
+    } else {
+      button = (
+        <Button
+          icon={{
+            name: "arrow-forward",
+            size: 15,
+            color: "white"
+          }}
+          title="Directly to chatbot"
+          onPress={() => navigation.navigate("Home")}
+        />
+      );
+    }
```

Comment:
Nit:

For future reference for this, you can use a [tenary operator](https://guide.freecodecamp.org/c/ternary-operator/)
For example: 
```jsx
const button = !onBoarded ? ( 
       <Button
          icon={{
            name: "arrow-forward",
            size: 15,
            color: "white"
          }}
          title="Get started"
          onPress={() => navigation.navigate("Intro")}
        />)  
      :
       (<Button
          icon={{
            name: "arrow-forward",
            size: 15,
            color: "white"
          }}
          title="Directly to chatbot"
          onPress={() => navigation.navigate("Home")}
        />)


```